### PR TITLE
Translation to Arabic: 3 files

### PR DESCRIPTION
**Pull request by celyes**
Translated 3 files to arabic: 

`about/working-groups.md`
`site.json`
`404.md`

@Aissaoui-Ahmed  @InterstellarClub

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-ar：You can have a review, thanks!

File: locale/ar/about/working-groups.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+* [nodejs-ro - Romanian (Română)](https://github.com/nodejs/nodejs-ro)
+* [nodejs-ru - Russian (Русский)](https://github.com/nodejs/nodejs-ru)
+* [nodejs-sv - Swedish (Svenska)](https://github.com/nodejs/nodejs-sv)
+* [nodejs-ta - Tamil (தமிழ்)](https://github.com/nodejs/nodejs-ta)
+* [nodejs-tr - Turkish (Türkçe)](https://github.com/nodejs/nodejs-tr)
+* [nodejs-zh-TW - Taiwanese (國語)](https://github.com/nodejs/nodejs-zh-TW)
+* [nodejs-uk - Ukrainian (Українська)](https://github.com/nodejs/nodejs-uk)
+* [nodejs-vi - Vietnamese (Tiếng Việt)](https://github.com/nodejs/nodejs-vi)
+
+### [الإصدارات](https://github.com/nodejs/LTS)
```
Comment (ouadie-lahdioui) [inline locale/ar/about/working-groups.md]: LTS > الإصدارات الطويلة الأمد

File: locale/ar/about/working-groups.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+* تولي المسؤولية عن التوثيق الخاص بمشروع النود جي اس و تعديله.
+* مراجعة التغييرات للكلاسات الفرعية الخاصة بالتدفق ضمن مشروع النود جي اس.
+* إعادة توجيه التغييرات للتدفقات من مشروع النود جي اس إلى هذا المشروع.
+* المساعدة في تطبيق موفري التدفق ضمن النود جي اس.
+* التوصية بإدراج نسخ الـ `readable-stream` ضمن النود جي اس.
+* التراسل حول مستقبل التدفقات لإعطاء المجتمع لمحة مسبقة عن التغييرات.
+
+### [الموقع](https://github.com/nodejs/nodejs.org)
+
+تهدف مجموعة العمل الخاصة بالموقع إلى بناء و صيانة الموقع المتاح للعموم الخاص بمشروع النود جي اس.
```
Comment (ouadie-lahdioui) [inline locale/ar/about/working-groups.md]: بناء و صيانة  > تطوير وصيانة

**Review by ouadie-lahdioui — COMMENTED** [review]
- Cannot just let (النود جي اس) Node.js without translation ? it's a key world such C++, i18n
- I think this phrase مسؤليات المجموعة تشمل is more  significant than تشمل المسؤوليات 

WDYT ?

File: locale/ar/about/working-groups.md (hunk 1 above)
Comment (celyes) [reply locale/ar/about/working-groups.md]: We used the term "تشمل مسؤولياتها" to be exact

File: locale/ar/about/working-groups.md (hunk 2 above)
Comment (celyes) [reply locale/ar/about/working-groups.md]: ok

File: locale/ar/about/working-groups.md (hunk 1 above)
Comment (celyes) [reply locale/ar/about/working-groups.md]: I've corrected the mistake ... push process in progress

**Review by ouadie-lahdioui — APPROVED** [review]
LGTM 👍 

Thank u for the great work

**Pull request merged**